# ğŸ‘‹ Hi, I'm Paula 

Welcome to my GitHub! Iâ€™m a molecular and cellular biologist with over 20 years of experience in academic research and recent expertise in applied data science and machine learning. I combine deep scientific knowledge with modern analytical tools to tackle complex problemsâ€”particularly in biology and biomedicineâ€”but my passion extends beyond these fields. Iâ€™m driven by the power of big data and eager to contribute to data-driven projects that deliver insights, solutions, and impactful results across various domains.

## View my CV and Full Publication List:
Papers using bioinformatics,  molecular biology, genetics and cell biology approaches to study various aspects of RNA biology, developmental biology, the cell cycle, and cancer.

[ğŸ“„ View My CV and full publications list](https://paulavazq.github.io/CV/)

---

## ğŸ§ª Data Science and Machine Learning Projects 

Hereâ€™s a selection of personal, academic, and study-related projects that reflect my journey into applied data science:

- ğŸ«€ğŸ“ˆ [**ECG Anomaly detection with Autoencoders**](https://github.com/paulavazq/ecg-anomaly-detection-autoencoders)
   *  This project applies deep learning autoencoders to ECG signals, enabling anomaly detection and latent space visualization for improved understanding of cardiac patterns.
   * Tools:* PyTorch, wfdb, neurokit2, scikit-learn, matplotlib, Optuna

- ğŸ§¬ [**RNA-seq Prostate Cancer Clinical Outcome Analysis**](https://github.com/paulavazq/Final_Project_RNAseq)  
   * Identified clinical biomarkers using RNA-seq data, applying differential gene expression analysis and machine learning models (Logistic Regression, Random Forest, CNN on tabular data converted to images).
   * Tools:* Python, pandas, scikit-learn (Logistic Regression, Random Forest), TensorFlow/PyTorch, biopython, matplotlib


- ğŸµ [**Audio Classification with Deep Learning**](https://github.com/paulavazq/Module-6_Music-Speach) 
  * Developed CNN-based classifier for urban sound datasets classification (audio vs speach) using spectrograms, CNNs, and PyTorch *
  * Tools:* PyTorch, librosa, scikit-learn, matplotlib

- ğŸ’ŠğŸ“ˆ [**Drug Sensitivity Prediction (IC50)**] *(coming soon)*
     * Predicting drug sensitivity (measured as IC50 values) based on cell line and drug features.  
     * Comparing the performance of various machine learning models, including:  
      Linear Regression, Random Forest, Tuned K-Nearest Neighbors, Ridge Regression, and Feed Forward Networks (FF).
     * Tools:* scikit-learn, pandas, matplotlib, Optuna optimization, Keras (via TensorFlow)
    

- ğŸ‘ï¸ ğŸ¶ **Computer Vision Internship Project** *(in progress, private)*  
    *Behavioral tracking of dogs using deep learning object detection and pose estimation.*
    *Tools:* YOLO, DeepLabCut, OpenCV, Python

  
- ğŸ¬ğŸ©¸ **Diabetes Prediction with Machine Learning**  
    * Built supervised models to predict diabetes risk from women patient health data (classification)*
    *Tools:* Python, pandas, scikit-learn (Logistic Regression, XGBoost), matplotlib

---

## ğŸ“š Background Snapshot

- **Data Scientist**, DIRU, ZÃ¼rich University (2024â€“2025)  
  *Applied deep learning to behavioral imaging.*
  
- **Staff Scientist**, Bern University (2012â€“2023)  
  *Led multi-omics research (RIP-Seq, proteomics, genetics) and supervised MSc/PhD students.*

- **PhD in Natural Sciences**, Max Planck Institute / Kassel University (2004)  
  *Cap-independent translation in Drosophila; summa cum laude.*

- **CAS in Applied Data Science**, Bern University (2023â€“2024)  
  *Specialized in machine learning, bioinformatics, and Python/R workflows.*

---

## ğŸ§  Core Skills

**ğŸ“Š Data Science / ML**

- **Python:** Pandas, NumPy, matplotlib, seaborn, plotly, scikit-learn, TensorFlow, PyTorch  
- **R:** ggplot2, bioinformatics packages  
- **Other tools:** SQL, ImageJ/Fiji, GraphPad, CVAT labeling software, Jupyter Notebooks, Git
- **Machine learning techniques:** Classification, regression
- **Time Series Analysis:** Feature extraction, anomaly detection, and visualization for sequential data (ECGs, audio)
- **Deep learning & computer vision:** Autoencoders, CNNs, diffusion models


** ğŸ§ª Bio & Lab**
- **Transcriptomics:** RNA-seq, RIP-seq, RT-qPCR, NGS  
- **Proteomics:** In vitro translation, protein expression & purification  
- **Imaging:** Confocal microscopy, immunofluorescence  
- **Experimental models:** Drosophila, insect and mammalian cells  
- **Genome editing:** CRISPR/Cas9, fly genetics

---

## âœï¸ Publications

Lead or co-author on peer-reviewed publications in journals including *eLife*, *Development*, *J. Cell Sci*, and *PLoS Genetics*.  
Topics span RNA transport, translational control, tubulin modifications, and gene regulation.

See full list â†’ [CV & Publications](https://paulavazq.github.io/CV/)

---

## ğŸ“¬ Letâ€™s Connect

Iâ€™m open to collaborations, data-driven research projects in all disciplines.

ğŸ“§ paulavazq@gmail.com  
ğŸ“ Bern, Switzerland  
ğŸŒ [Website / CV](https://paulavazq.github.io/CV/)





